modules/sdk-coin-baby/src/lib/EpochedStakingActivateBuilder.ts

```diff
@@ -0,0 +1,11 @@
+import * as constants from './constants';
+import { StakingActivateBuilder } from '@bitgo/abstract-cosmos';
+
+export class EpochedStakingActivateBuilder extends StakingActivateBuilder {
```

Comment:
why are you defining a new class for each builder ?

Reply:
The Baby delegate/undelegate/switch validator messages are wrapped messages of the corresponding cosmos ones so we need to set the type url here to differentiate between them.